go-controller/pkg/ovn/policy.go

```diff
@@ -358,6 +358,12 @@ func (oc *Controller) getMulticastACLMatch(ns string) string {
 	return "ip4.src == $" + nsAddressSet + " && ip4.mcast"
 }
 
+// Returns the multicast port group name and hash for namespace 'ns'.
+func (oc *Controller) getMulticastPortGroup(ns string) (string, string) {
```

Comment:
@dceara don't need `(oc *Controller)` here since it doesn't use any members of `oc`

Reply:
@dcbw, ok, fixed and also rebased on the latest master.